images-list

```diff
@@ -369,6 +369,7 @@ k8s.gcr.io/sig-storage/livenessprobe rancher/mirrored-sig-storage-livenessprobe
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 0.1.151
 kiwigrid/k8s-sidecar rancher/kiwigrid-k8s-sidecar 1.1.0
 kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 0.1.151
+library/bash rancher/mirrored-library-bash 5.1.4
```

Comment:
This image, and bash in general, is [licensed as GPL3](https://hub.docker.com/_/bash). Before merging we would need to confirm that this specific image is fine to incorporate into the product.

Reply:
We are checking internally about this (at the moment I'm waiting to confirm some meetings).

If only bash is needed, I would recommend to use [bci/bci-minimal](https://registry.suse.com/static/bci/bci-minimal-15sp4/index.html) instead.

Reply:
Discussion running slack.

Reply:
Currently trialing a replacement which should not have legal issues. See https://github.com/epinio/epinio/pull/1741 and https://github.com/epinio/helm-charts/pull/276

Reply:
Ah, we crossed here @macedogm  GH did not show me your comment when you made it. Might have shaved a bit of time ... @pgonin also brought BCI up and from there the research/work got started.

Reply:
Current plan is to have the image as `ghcr.io/epinio/epinio-unpacker:TAG-TO-BE-DETERMINED`

Reply:
Tag is 1.0. Force pushed new commit using `ghcr.io/epinio/epinio-unpacker:1.0`.

THE PR https://github.com/rancherlabs/eio/issues/1309 is still the necessary EIO request.